### PR TITLE
Changed all abilitys which do dmg or drop units.

### PR DESCRIPTION
**Pull request by trempler**
Duration/Damage/Behaviour

### CODE REVIEW

File: rules/feature/targettype.yaml [hunk 1]
```diff
@@ -96,3 +96,11 @@
 		RequiresForceFire: true
 	Targetable@TargetTypeNoAutoTarget:
 		TargetTypes: NoAutoTarget
+
+^FeatureTargettypeFloraAir:
```
Comment (jrb0001) [inline rules/feature/targettype.yaml]: This name doesn't make any sense at all. Why do you need it?

File: rules/feature/targettype.yaml [hunk 2]
```diff
@@ -96,3 +96,11 @@
...
+	Targetable@Targettype:
+		TargetTypes: Flora
+		-RequiresCondition:
```
Comment (jrb0001) [inline rules/feature/targettype.yaml]: No overrides please.

File: rules/gdi-structures.yaml [hunk 3]
```diff
@@ -405,8 +405,8 @@ GAARMORY:
 		OrderName: ArtilleryStrikePowerInfoOrder
 		Prerequisites: garadr, upgrade.gdf
 		Icon: artystrike
-		ChargeTime: 270
-		QuantizedFacings: 8
+		ChargeTime: 330
+		QuantizedFacings: 2
```
Comment (jrb0001) [inline rules/gdi-structures.yaml:409]: Only two directions, are you sure? This applies to all of them.

**Review by jrb0001 — CHANGES_REQUESTED** [review]
I think reducing the possible flight angles for airstrike abilities is the wrong way. This just makes it much easier to build defences against them while not solving any issue.

File: rules/gdi-structures.yaml (hunk 3 above)
Comment (trempler) [reply rules/gdi-structures.yaml:409]: This was discused with Ganon in a Voice and yes i am sure, to make abilitys less random and more tactical. If the change sucks, we can still revert it.

File: rules/feature/targettype.yaml (hunk 1 above)
Comment (trempler) [reply rules/feature/targettype.yaml]: I used it to stop turrets and units from targeting the Units. Feel free to rename it if you dislike it.

File: rules/gdi-structures.yaml (hunk 3 above)
Comment (jrb0001) [reply rules/gdi-structures.yaml:409]: Predicable abilities have a small advantage for the attacker and a big advantage for the defender. I don't really see the point of doing this change but let's try it.

File: rules/feature/targettype.yaml (hunk 1 above)
Comment (jrb0001) [reply rules/feature/targettype.yaml]: Then you probably want the `^FeatureTargettypeNoAutoTarget` one which disables auto target (force fire is still possible though). If you don't want them to be targeted at all, just remove those Inherits completely.

Comment (trempler) [reply rules/feature/targettype.yaml]: Well I did not know about this, I thought it would crash if you have none selected.

Comment (jrb0001) [reply rules/feature/targettype.yaml]: So you want to have them to be completely untargetable?

Comment (trempler) [reply rules/feature/targettype.yaml]: Yes

Comment (jrb0001) [reply rules/feature/targettype.yaml]: fixed.

File: rules/feature/targettype.yaml (hunk 2 above)
Comment (jrb0001) [reply rules/feature/targettype.yaml]: fixed.

**Review by jrb0001 — APPROVED** [review]

**Pull request merged**